### ISSUE TITLE
Plan screen - Revised button layout

### ISSUE THREAD

**Issue by EVGrokker** [aesculus/EVTO-App-Feedback#328]
I'm proposing rearranging the buttons on the Plan screen as shown in the attached image. Here's the rationale:

We currently have 5 buttons:
- Globe: recenters map on active segment or entire trip
- Folded Map: Opens Segment Summary
- Location: Toggles device location, recenters map upon toggling ON
- Lighting bolt: Toggles display of superchargers
- Outlet: Toggles display of other chargers.

I've added a horizontal grid line in the image to show the approximate vertical halfway point for reference. 

My thinking:

Buttons in the upper half deal with selecting, managing and displaying the active route. Some operations change the zoom or centering of the map.

Buttons in the lower half deal with toggling overlays.

The Location button both recenters the map and toggles an overlay of the device location, so it's in the middle.

By grouping the related buttons in the upper and lower areas, that gives us room to add more buttons in the future if needed, without disrupting the user's sense of how they're organized.

This was a quick hack, so excuse the imprecise locations and subdivisions.

The verbal description of the new button location would be as follows:

The Location button should be centered vertically in the right-side button stack.

The Globe button and Map buttons should be grouped either side of an imaginary line 25% down the screen, or the middle of the top half. 

The Lighting bolt and Outlet buttons should be grouped either side of an imaginary line 75% down the screen, or the middle of the bottom half.

![button layout 2](https://cloud.githubusercontent.com/assets/26678403/25435149/56ff48d4-2a4c-11e7-8dee-d87e16519fa6.png)

**Comment by aesculus** [comment]
I think I can get close to this with a bit of hacking.  But let me interject a bit of facts in here just to muddle the situation.

Google defines 12 sectors for placing button like these on the map. I can then tweak these by adding padding etc between controls that share the same position.

+----------------+ 
  + TL    TC    TR + 
  + LT          RT + 
  +                + 
  + LC          RC + 
  +                + 
  + LB          RB + 
  + BL    BC    BR + 
  +----------------+

**Comment by EVGrokker** [comment]
Give it a (w)hack.  You can just send a screen shot when you have something worth looking at.

**Comment by EVGrokker** [comment]
If you can't push things to the middle of the upper half and the middle of the lower half, I'd be OK with the current buttons being pinned to the top (Globe, Map) and bottom (Supercharger, Outlet), with Location in the middle.

**Comment by EVGrokker** [comment]
Visually, this would be OK too:

![button layout 3](https://cloud.githubusercontent.com/assets/26678403/25444176/726e1fee-2a67-11e7-810a-651703f57279.png)

**Comment by aesculus** [comment]
That's what I am going to do probably. But I can add a lot of spacing at the top (globe) which should give it the same effect.

Another piece of info that I am surprised has not shown up yet on iOS, is the addition of a button to go full screen map on Androids.

![screenshot_2017-04-26-09-02-35](https://cloud.githubusercontent.com/assets/283364/25444240/4af051ec-2a5f-11e7-8ad4-bdd0ef1d9b2a.png)

**Comment by aesculus** [comment]
See if this works. V1.2 (7)

Space at the top for the Android full screen map button.

![image](https://cloud.githubusercontent.com/assets/283364/25456548/13f60712-2a88-11e7-8d77-c0a95e9c8cdf.png)

**Comment by EVGrokker** [comment]
Looks good to me!

**Comment by aesculus** [comment]
Actually I hated it after using it.  I went with this layout for the following reasons:
- The three most used buttons for me are the globe, map and lightening bolt - I like them visually separate
- What you cannot see in the above picture is how bad the red circle interferes with the globe icon, both visually and spatially.
- My mind is trained to go to the center for the map
- I think the grouping more logical too since the first two buttons actually change whats on the map. The map icon brings up a dialog.

![image](https://cloud.githubusercontent.com/assets/283364/25458038/31254848-2a8d-11e7-862a-5bf5ad86b69e.png)
